CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Snyk Security - Code and Open Source Dependencies Changelog
 
+## [1.2.10]
+
+### Added
+
+- Encryption for when storing the Snyk token after successful login.
+
+### Changed
+
+- Extension name to "Security - Code and Open Source Dependencies".
+
+### Removed
+
+- The token text field from the extension configuration and will not be visible anymore.
+
```

Comment:
### Added
Surface request ID when Snyk Code analysis fail in the output channel.

### Fixed
"Illegal argument: character must be non-negative" error upon receiving Snyk Code analysis.